tcf_website/templates/reviews/edit_review.html

```diff
@@ -9,7 +9,7 @@
 
 {% block content %}
 <div class="new-review container text-left mx-auto">
-  <h1 class="h2 mt-2 mb-4">Edit a review</h2>
+  <h1 class="h2 mt-2 mb-4">Edit a review</h1>
```

Comment:
I'm pretty sure it's already been established why we need `<h1>` tags with `<h2>` classes, but I just find it funny